src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/FileStorageService.cs

```diff
@@ -75,7 +75,7 @@ public async Task CopyReleaseToPublicContainer(PublishReleaseDataMessage message
                 (destination as CloudBlockBlob)?.Metadata.Add("releasedatetime", releasePublishedString);
             };
 
-            await TransferManager.CopyDirectoryAsync(sourceDirectory, destinationDirectory, false, options, context);
+            await TransferManager.CopyDirectoryAsync(sourceDirectory, destinationDirectory, true, options, context);
```

Comment:
This Azure 'service copy' appears to fix the problem not copying the existing meta attributes (bug with DFE-871 Publishing Function). When using the service copy, meta attributes are copied.